### DISCUSSION TITLE
Multiple passwords

### DISCUSSION THREAD

**Discussion by LoranRendel** [rustdesk/rustdesk#2912] [General]
Please add capability to set multiple passwords for different users.

**Comment by CryptoZyphix** [comment]
Yes please add multiple password capability on the client directly in the settings. This could also be added to the Strategies menu so that it will auto-push out to all clients vs re-building a custom client and then re-deploying on each PC. I think in this case the best option would be through Strategies. 

Also please add the ability not to require a password when connecting to clients if the user is connecting from a RustDesk Pro account from the Web Console, then it will identify you with your Web Console account through the API. To add more security I would only allow the connection in this way if the user has 2FA enabled otherwise it should prompt them for the client password. TeamViewer has this implemented already they call it "Ease of Use" They also already have the ability to add multiple passwords directly on the client side on a per-client basis.
